utils/get_opt_externals

```diff
@@ -22,26 +22,28 @@ Downloads optional external components.
 
 Possible choices for OPTEXT:
 
-  DISTRIB (default) -- download only components which are LGPL-compatible,
-      can be distributed with the DFTB+ source code and do not restrict the
+  UNRESTRICTED (default) -- download only components which are less or equally
+      restrictive than the LGPL license of DFTB+ and do not restrict the
       distribution of the compiled code beyond its LGPL-license.
 
+  RESTRICTED -- download optional components with stricter license as LGPL
+      (typically GPL). The distribution of the compiled code will be restricted
+      beyond the LGPL license and governed by the strictest license of the
+      included optional components (typically GPLv3).
+
   ALL -- download all optional external components.
 
-      NOTE: this will also download components with LGPL-incompatible
-      licenses. Those components can not be distributed with the DFTB+ source
-      code and the distribution of the resulting binary or library is not
-      permitted if those components were part of the build. Using them for
-      personal research is permitted, though.
+      NOTE: this will also download components which may restrict the license of
+      the compiled code beyond the LGPL license.
```

Comment:
LGPL3

Reply:
I made it now more explicit at several places.